app/controllers/events/routes.py

```diff
@@ -5,6 +5,28 @@
 from app.controllers.events import events_bp
 from app.logic.events import getEvents
 from app.logic.participants import sendUserData
+from datetime import datetime
+from app.models.eventRsvp import EventRsvp
+
+@events_bp.route('/events/<term>/', methods=['GET'])
+def events(term):
+    #set term to current term when events page is accessed from the navbar
+    if not term.isdigit():
+        term = g.current_term
+
+    currentTime = datetime.now()
+    eventsDict = groupEventsByCategory(term)
+    listOfTerms = Term.select()
+    participantRSVP = EventRsvp.select().where(EventRsvp.user == g.current_user)
+    rsvpedEventsID = [event.event.id for event in list(participantRSVP)]
```

Comment:
I don't think you need the `list()` call here. Query objects are iterable.

Reply:
This route is no longer used.